tests/apple-exporter/components/test-class-quote.php

```diff
@@ -203,6 +204,88 @@ public function testSettingsBlockquote() {
 			10,
 			$json['components'][1]['style']['border']['all']['width']
 		);
+		$this->assertFalse( isset( $json['components'][1]['style']['conditional'] ) );
+	}
+
+	/**
+	 * Test Dark Color Settings For Blockquote.
+	 *
+	 * @access public
+	 */
+	public function testDarkColorsBlockquote() {
+		// Setup.
+		$content = new Exporter_Content(
```

Comment:
Alternately, you could use the new function `get_json_for_post` which should simplify a lot of this setup: https://github.com/alleyinteractive/apple-news/blob/feature/v2.1.0/tests/apple-exporter/builders/test-class-advertising-settings.php#L116-L136

For example, you could do something like this:
```php
$post_id = self::factory()->post->create(
	[
		'post_content' => '<blockquote><p>my quote</p></blockquote>',
	]
);
$json    = $this->get_json_for_post( $post_id );
$this->assertEquals(
	'#abcdef',
	$json['componentTextStyles']['default-blockquote-left']['conditional']['textColor']
);
$this->assertEquals(
	'#fedcba',
	$json['components'][1]['style']['conditional']['backgroundColor']
);
$this->assertEquals(
	'#012345',
	$json['components'][1]['style']['conditional']['border']['all']['color']
);
```

You may need to adjust the JSON references, as the full article JSON might be a little different from just running it through `Exporter`.